upgrades/gov-props/v1.2.0.md

```diff
@@ -0,0 +1,24 @@
+# v1.2.0 - Bouncy Bulldog
+
+## Summary
+This upgrade allows the Jackal Network to start fulfilling its purpose as the Decentralized Storage Platform of the Cosmos. This upgrade introduces the Storage module, which allows users to pay for storage and create storage deals with Storage Providers. It also brings the addition of the Filetree module which creates a privacy layer and bucket file structure akin to S3 and Google Drive. Finally it also brings the addition of the Oracle module which allows other modules to query a data stream being supplied by a centralized entity. In this case, Jackal Labs will be using this oracle to supply pricing information while the AMM module is being finalized. 
+## Change Log
+* Re-added Storage Module
+* Re-added Filetree Module
+* Added Oracle Module
+* Fixed RNS marketplace bugs
+* Increased Code Test Coverage by ~60%
```

Comment:
Erin said it should be 'Coverage to ~60%' instead of 'by'?

Reply:
Yeah I should change it to `Coverage to ~50%`